Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-platform :ios, '10.0'
+platform :ios, '11.0'
```

Comment:
Is there a reason for this? I thought still target iOS 10 minimum for the whole platform.

Reply:
FutureNova minimum is 11.0 :(